serde_codegen/src/ser.rs

```diff
@@ -57,12 +57,43 @@ pub fn expand_derive_serialize(
     );
 
     let where_clause = &impl_generics.where_clause;
+    /*
```

Comment:
:) [this](https://github.com/serde-rs/serde/pull/208#issuecomment-172601055) should help with that.